### PR TITLE
Applied feedback

### PR DESCRIPTION
**Pull request by Kaaserne**

### CODE REVIEW

File: code/headers/mlx90640_i2c.hpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #pragma once
 #include <i2c_bus.hpp>
-
+#include <registers.hpp>
 namespace r2d2::thermal_camera {
```
Comment (itzandroidtab) [inline code/headers/mlx90640_i2c.hpp:5]: You are missing a newline between the namespace and the include

File: code/headers/mlx90640_i2c.hpp [hunk 2]
```diff
@@ -21,14 +21,15 @@ namespace r2d2::thermal_camera {
...
+        uint16_t read_register(const registers internal_address) const;
 
         /**
          * Writes a register.
          *
          * @param uint16_t internal_address of the chip.
-         * @param uint16_t data to be written*/
-        void write_register(const uint16_t internal_address,
+         * @param uint16_t data to be written
+         * */
```
Comment (itzandroidtab) [inline code/headers/mlx90640_i2c.hpp]: The `*` before the `*/` should be removed. The other doxygen have this as well

File: code/src/mlx90640_i2c.cpp [hunk 3]
```diff
@@ -5,19 +5,22 @@ namespace r2d2::thermal_camera {
         : bus(bus), address(address) {
     }
 
-   uint16_t mlx90640_i2c_c::read_register(const uint16_t internal_address) 
-                                          const {
+
```
Comment (itzandroidtab) [inline code/src/mlx90640_i2c.cpp:8]: This newline is not needed

File: code/src/mlx90640_processor.cpp [hunk 4]
```diff
@@ -43,47 +40,51 @@ namespace r2d2::thermal_camera {
...
-        int ram_vdd_pix = bus.read_register(RAM_VDD_PIX);
+        int ram_vdd_pix = bus.read_register(registers::RAM_VDD_PIX);
         apply_treshold(ram_vdd_pix, 32767, 65536);
 
         uint8_t res_cor = get_resolution_correlation();
         float Vdd = ((res_cor * ram_vdd_pix - Vdd25) / Kvdd) + VDD0;
         return Vdd;
     }
 
+    /* Page 36 to 37 */
```
Comment (itzandroidtab) [inline code/src/mlx90640_processor.cpp]: What is on page 36/37?

File: code/src/mlx90640_processor.cpp [hunk 5]
```diff
@@ -92,12 +93,12 @@ namespace r2d2::thermal_camera {
 
         return Ta;
     }
-
+    /* Page 37 */
```
Comment (itzandroidtab) [inline code/src/mlx90640_processor.cpp]: Same as other comment

File: code/headers/registers.hpp [hunk 6]
```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+/**
+ * MLX90640 16 bits registers
+ * INTERNAL means internal register.
+ * EE means EEPROM (Electric Erasable Programmable Read-Only)
+ * RAM means just RAM (Random Acces Memory)
+ **/
```
Comment (itzandroidtab) [inline code/headers/registers.hpp]: Same as the other * */

**Review by itzandroidtab — CHANGES_REQUESTED** [review]

File: code/src/mlx90640_i2c.cpp (hunk 3 above)
Comment (MarcDirven) [reply code/src/mlx90640_i2c.cpp]: It is, or else we'll get this:
```cpp
uint16_t
mlx90640_i2c_c::read_register(const uin16 internal_address) const {
```

**Review by itzandroidtab — APPROVED** [review]

**Pull request merged**